sql/reference/01-buildings_reference_schema.sql

```diff
@@ -0,0 +1,129 @@
+------------------------------------------------------------------------------
+-- Create buildings reference schema and tables
+------------------------------------------------------------------------------
+
+SET client_min_messages TO WARNING;
+
+
+CREATE SCHEMA IF NOT EXISTS buildings_reference;
+
+-- Suburb / Locality
+
+CREATE TABLE IF NOT EXISTS buildings_reference.suburb_locality (
+      suburb_locality_id serial PRIMARY KEY
+    , external_suburb_locality_id integer
+    , suburb_4th character varying(60)
+    , suburb_3rd character varying(60)
+    , suburb_2nd character varying(60)
+    , suburb_1st character varying(60)
+    , shape public.geometry(MultiPolygon, 2193)
+);
+
+-- Town / City
+
+CREATE TABLE IF NOT EXISTS buildings_reference.town_city (
+      town_city_id serial PRIMARY KEY
+    , external_city_id integer
+    , name character varying(60)
+    , shape public.geometry(MultiPolygon, 2193)
+);
+
+-- Territorial Authority
+
+CREATE TABLE IF NOT EXISTS buildings_reference.territorial_authority (
+      territorial_authority_id serial PRIMARY KEY
+    , external_territorial_authority_id integer
+    , name character varying(100)
+    , shape public.geometry(MultiPolygon, 2193)
+);
+
+-- Coastline
+
+CREATE TABLE IF NOT EXISTS buildings_reference.coastlines (
+      coastline_id serial PRIMARY KEY
```

Comment:
Add an `external_coastline_id integer` and equivalent for the other tables. We'll store the `t50_fid` there in order to detect which rows need to be updated when changes occur to the source.